charts/rpe-send-letter-service/requirements.yaml

```diff
@@ -1,4 +1,4 @@
 dependencies:
   - name: java
-    version: ~2.16.0
+    version: ~2.18.0
```

Comment:
should we not do this separately we are adding too many changes in one PR

Reply:
this one doesn't do any harm. the prep'ed beta v3 - does. since chart version needs a bump anyway, might as well update this. in other projects we already bumped from various stages including 2.15 and all that is a non-break stuff, just having extra feature, nicer handles in tests etc